### PR TITLE
CLOUDP-220794: add required instance flag to help usage in connection list

### PR DESCRIPTION
**Pull request by pulkitkalra-mdb**
## Proposed changes
[CLOUDP-220794](https://jira.mongodb.org/browse/CLOUDP-220794)

The `--instance` flag is required for the connections streams commands so its desirable to display it in the usage docs when accessing `-h` for a command. This PR updates the `Use` to add that in.

## Checklist
[✅ ] Ran make gen-docs

### CODE REVIEW

**Review by jarjee — APPROVED** [review]
LGTM

**Review by gssbzn — CHANGES_REQUESTED** [review]
we don't add the flags to usage

File: docs/atlascli/command/atlas-streams-connections-create.txt [hunk 1]
```diff
@@ -41,6 +41,10 @@ Arguments
      - string
      - false
      - Name of the connection
+   * - instance_name
+     - string
+     - true
+     - Name of the instance.
```
Comment (gssbzn) [inline docs/atlascli/command/atlas-streams-connections-create.txt:47]: this is listed under options 
```
 * - -i, --instance
     - string
     - true
- Name of your Atlas Stream Processing instance.
```
by adding it here it make the docs confusing

**Comment by pulkitkalra-mdb** [comment]
Thanks for your input @gssbzn, after discussing with the team I'm going to close this since it breaks the cli help usage guidelines.

**Pull request closed without merging**